src/utils/cryptography.ts

```diff
@@ -1,7 +1,7 @@
 import CryptoJS from "crypto-js";
 
 const encrypt = (file: string, secret: string): CryptoJS.lib.CipherParams => {
-    return CryptoJS.AES.encrypt(file, secret);
+  return CryptoJS.AES.encrypt(file, secret);
```

Comment:
can we agree on the linting rules?

Reply:
This was just from applying the project's `yarn lint`, which should be the only linting rules we have, I think?

Reply:
Is something linting / formatting it differently on your side?

Reply:
I think my vsc prettier plugins clashes with the project. I will disable it

Reply:
Hmm, is it [esbenp.prettier-vscode](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode)? That's supposed to use the project's prettier config by default… if it isn't, there may be something else going on?

It would be good to get automatic formatting sorted out, either through the plugin, or through a project pre-commit hook.

Reply:
yes it is